test/ngAnimate/animateCssSpec.js

```diff
@@ -28,9 +35,19 @@ describe("ngAnimate $animateCss", function() {
 
   var ss, prefix, triggerAnimationStartFrame;
   beforeEach(module(function() {
-    return function($document, $window, $sniffer, $$rAF, $animate) {
+    return function($document, $sniffer, $$rAF, $animate) {
       prefix = '-' + $sniffer.vendorPrefix.toLowerCase() + '-';
-      ss = createMockStyleSheet($document, $window);
+      ss = createMockStyleSheet($document);
+      ss.addPossiblyPrefixedRule = function(selector, styles) {
```

Comment:
I thought about that too. I chose to make the change locally, since it isn't currently used outside of that file, but I am fine putting it in `createMockStyleSheet()` if you think it's better.